front/app/modules/commercial/content_builder/admin/components/CraftComponents/Iframe/index.tsx

```diff
@@ -150,9 +150,10 @@ Iframe.craft = {
 const isValidUrl = (url: string) => {
   // Used this reference for generating a valid URL regex:
   // https://tutorial.eyehunts.com/js/url-regex-validation-javascript-example-code/
-  let invalidUrl = !url.match(
-    /(http(s)?:\/\/.)?(www\.)?[-a-zA-Z0-9@:%._+~#=]{2,256}\.[a-z]{2,6}\b([-a-zA-Z0-9@:%_+.~#?&//=]*)/g
-  );
+  const validUrlRegex = [
+    /(http(s)?:\/\/.)?(www\.)?[-a-zA-Z0-9@:%._+~#=]{2,256}\.[a-z]{2,6}\b([-a-zA-Z0-9@:%_+.~#?&//=]*)/g,
+  ];
+  let invalidUrl = !validUrlRegex.some((rx) => rx.test(url));
```

Comment:
Why do you put 1 regex in an array and use `some`? Do you expect to add more array elements later on? If not, I would write:

```
const validUrlRegex = /(http(s)?:\/\/.)?(www\.)?[-a-zA-Z0-9@:%._+~#=]{2,256}\.[a-z]{2,6}\b([-a-zA-Z0-9@:%_+.~#?&//=]*)/g
let invalidUrl = !validUrlRegex.test(url);
```

Reply:
Oh ok! Just a bit of a misunderstanding on my part. I'll update that :+1:  Thanks!